pkg/validation/limits.go

```diff
@@ -406,6 +412,14 @@ func (o *Overrides) UnorderedWrites(userID string) bool {
 	return o.getOverridesForUser(userID).UnorderedWrites
 }
 
+func (o *Overrides) MaxLocalStreamRateMB(userID string) int {
+	return o.getOverridesForUser(userID).MaxLocalStreamRateBytes.Val()
+}
+
+func (o *Overrides) MaxLocalStreamBurstRateMB(userID string) int {
+	return int(o.getOverridesForUser(userID).MaxLocalStreamBurstRateBytes)
+}
+
```

Comment:
Why are these calculated in different ways (`int(x)` vs `x.Val()`)? Let's consistently use `.Val()`.